includes/wikia/tests/WikiaDispatchableObjectTest.php

```diff
@@ -132,8 +139,11 @@ public function testCheckWriteRequest( $params, $wasPosted, $isInternal, $token,
 			->disableOriginalConstructor()
 			->getMockForAbstractClass();
 
+		$app = new WikiaApp( new WikiaLocalRegistry() );
+		$app->wg->User = $userMock;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)